clients/withManifest.ts

```diff
@@ -360,10 +368,14 @@ export const proxy = <TManifest extends AppManifest>(
   return proxyFor(invoke<TManifest>(fetcher) as typeof batchInvoke);
 };
 
+interface forAppType<TApp extends App> {
```

Comment:
Is this type necessary? Isn't the same as `withManifest` ?

Reply:
I think the difference is the `InvocationProxyWithBatcher` that I put in order to satisfy the `proxy` return type.

Reply:
I don't think so, I just verified that you can use it